README.md

```diff
@@ -271,3 +274,35 @@ ENTRYPOINT ["/go-binary"]
 ### Build final docker image
 
 `docker build -t docker-edgex-binary -f Dockerfile --build-arg BASE=base-build .`
+
+## Deployment On Sandbox
+
+
+The official site for Sandbox environment is **[https://jenkins.edgexfoundry.org/sandbox/](https://jenkins.edgexfoundry.org/sandbox/)**
+
+#### Get Access To Sandbox
+
+The official process to request access sandbox is provided in the link **[Here](https://https://lf-releng-docs.readthedocs.io/en/latest/jenkins-sandbox.html#get-access-to-the-jenkins-sandbox)**
+
+If you already have the LFID and can access the Prod Environment you can use the same LFID to access the Sandbox.
+
+#### Implementation
+
+Once you create a new Jenkins file follow the procedure as above. In your Github repository, your jenkinsfile name should be Jenkinsfile.sandbox. Since, the jenkins sanbox will recognizes only the file Jenkinsfile.sandbox to execute.
```

Comment:
spelling - sandbox

Reply:
capitalize Jenkins

Reply:
i apologize i donno  missed this some how

Reply:
fixed